forwarding/terraform/webhook/forward-via-webhook.tf

```diff
@@ -14,7 +14,6 @@ resource "google_cloud_run_v2_service" "webhook-application" {
             version = "latest"
           }
         }
-        value = var.CUSTOMER_1_WEBHOOK_KEY
```

Comment:
not sure how/from where this variable was set, but I think it should not be needed in `late`

Reply:
Yes, that's a leftover from the "secrets via GHA" in mid (which was replaced by "secrets via GCP secret manager" in late). Thanks for the fix!